### PR TITLE
Companion pnp side

### PR DESCRIPTION
**Pull request by magnificus**

### CODE REVIEW

File: src/PropertyArrayContainer.tsx [hunk 1]
```diff
@@ -23,6 +23,7 @@ import Socket from './classes/SocketClass';
 import { SocketContainer } from './SocketContainer';
 import { CodeEditor } from './components/Editor';
 import InterfaceController, { ListenEvent } from './InterfaceController';
+import useInterval from 'use-interval';
```
Comment (fakob) [inline src/PropertyArrayContainer.tsx:26]: I don't see a change in the package.json. Is this package already included?

File: src/PropertyArrayContainer.tsx [hunk 2]
```diff
@@ -397,6 +413,14 @@ export const PropertyArrayContainer: React.FunctionComponent<
     });
   };
 
+  const [socketsCurrentlyRendered, setSocketsCurrentlyRendered] = useState(
+    getSocketsCurrentlyRendered(singleNode)
+  );
+
+  useInterval(() => {
+    setSocketsCurrentlyRendered(getSocketsCurrentlyRendered(singleNode));
+  }, 100);
```
Comment (fakob) [inline src/PropertyArrayContainer.tsx:422]: Do I understand correctly that every 100ms it checks and updates?

**Review by fakob — APPROVED** [review]

File: src/PropertyArrayContainer.tsx (hunk 1 above)
Comment (magnificus) [reply src/PropertyArrayContainer.tsx:26]: yes indeed

File: src/PropertyArrayContainer.tsx (hunk 2 above)
Comment (magnificus) [reply src/PropertyArrayContainer.tsx:422]: yes indeed :)

**Pull request merged**